res/qml/ComboBox.qml

```diff
@@ -12,9 +12,11 @@ ComboBox {
     delegate: ItemDelegate {
         id: itemDlgt
 
+        required property int index
+
         width: parent.width
-        highlighted: root.highlightedIndex === index
-        text: root.textAt(index)
+        highlighted: root.highlightedIndex === this.index
```

Comment:
Does this fix the various *property does not exist* warnings I saw when starting the Qt6 QML version?

Reply:
If you mean the "verticalCenter" warnings, then no, these are adifferent issue.

But this fixes a bunch of qmllint warnings where it complains about unqualified lookups.

https://doc.qt.io/qt-6/qml-codingconventions.html#required-properties